### ISSUE TITLE
[ENHANCEMENT]: Show Total File Size for Selected Files and Total Number of Items for Selected Directories

### ISSUE THREAD

**Issue by iamkartiknayak** [pop-os/cosmic-files#481]
**Cosmic-files version:**
```
cosmic-files:
  Installed: 0.1.0~1726676662~22.04~8668c18
  Candidate: 0.1.0~1726676662~22.04~8668c18
```

**Issue/Bug description:**
At present this feature only exist as a part of details pane. Cosmic Files needs this enhancement that displays the total file size when files are selected and the total number of items when a directory is selected. This functionality would provide users with quick insights into their selections.

**Steps to reproduce:**
N/A

**Expected behavior:**
- When multiple files are selected, the total size of the selected files should be displayed in the UI (preferably in bottom right).
- When a directory is selected, the total number of items (files and subdirectories) within that directory should be shown.

**Other notes:**
![screenshot-2024-09-20-14-10-22](https://github.com/user-attachments/assets/e561e903-4d0d-409d-8d17-2c2e4cdd676a)

**Comment by colino17** [comment]
Agreed. This is pretty basic file manager functionality that appears to be missing. I noticed it today on my test machine when I tried to select multiple files to see the total size and couldn't find a way to do it.

**Comment by iamkartiknayak** [comment]
Closing this issue as it's being tracked here: https://github.com/pop-os/cosmic-files/issues/481
